README.md

```diff
@@ -1,11 +1,11 @@
-# Hermione
+# Testplane (ex-Hermione)
 
-Hermione is a utility for integration testing of web pages using [WebdriverIO](https://webdriver.io/docs/api) and [Mocha](https://mochajs.org).
+Testplane is a utility for integration testing of web pages using [WebdriverIO](https://webdriver.io/docs/api) and [Mocha](https://mochajs.org).
 
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-- [Why you should choose hermione](#why-you-should-choose-hermione)
+- [Why you should choose testplane](#why-you-should-choose-testplane)
```

Comment:
Here in similar cases:

1) `Testplane` as product name must starts with uppercase letter T.
2) `testplane` as CLI name or command name must be lowercase letter.